documentation/azure-powershell-release-template.md

```diff
@@ -0,0 +1,88 @@
+# Azure PowerShell Release Notes
+
+The following list outlines the steps necessary for a successful release of Azure PowerShell. Please check off the boxes as you complete the steps.
+
+- [ ] For all existing PRs, run the **powershell-demand** Jenkins job on each one, and merge if the build is successful and has been signed off by a member of the team
+- [ ] Run the **powershell-demand** Jenkins job against the **dev** branch and fix any issues
+- [ ] In the Azure PowerShell repo, create a branch for the release and name it **release-X.X.X**
+- [ ] Restrict access to the release branch through the GitHub settings so partners cannot merge their PRs themselves
+    - In the settings window, find the *Protected branches* section, and select the release branch from the drop-down menu
+    - Make sure the following are the only things checked off:
+        + *Protect this branch*
+        + *Require status checks to pass before merging*
+        + *default*
+        + *Restrict who can push to this branch*
+            * Make sure to add *Azure/adx-sdk-manage* to this list of people with push access
+    - You can check the settings for the **dev** branch if you have any questions
+- [ ] Clone the Azure PowerShell repo to your computer, and switch to the release branch
+- [ ] Open a PowerShell window, and run the script **tools\PreparePSRelease.ps1**
+    - Check the script header for a sample of how the parameters should look
+    - For example, for the August 2016 release of Azure PowerShell 2.0.0, the command would look like the following:
+    > `.\PreparePSRelease.ps1 2.0.0 "August 2016" -Major`
+- [ ] Create a signed build from the release branch using the **powershell-sign** Jenkins job
+- [ ] Once the build is successful, click on *Build Artifacts* and download the zip
+    - This zip file will contain the .msi for Azure PowerShell, and all of its packages
+- [ ] Install Orca from the .msi in the following folder: **\\\\aaptfile01\adxsdk\PowerShell**
+- [ ] In the zip file, rename the .msi to **azure-powershell.X.X.X.msi**
+- [ ] Open this .msi in Orca, select the *Property* page in the left panel, and save the *ProductCode* somewhere you can easily access it
+- [ ]  Create an account on [CodePlex](codeplex.com) if you don't already have one, and link it with your Microsoft corporate account
+- [ ]  To access the [Web Platform Install Feeds](https://webpi.codeplex.com/) repository,  contact [Chris Sfanos](mailto:chris.sfanos@microsoft.com) with your CodePlex username and request to join the Web Platform Installer project
```

Comment:
I would put this preparatory step up higher, and caveat it that it needs to be done one time only,